migrations/20220519070647_arpa_recipients_table.js

```diff
@@ -0,0 +1,17 @@
+exports.up = function (knex) {
+  return knex.schema
+    .createTable('arpa_recipients', function (table) {
```

Comment:
Are "recipients" here a different concept from "subrecipients" as defined in the export template?

I'm thinking in the UI we're probably going to want to be clear that these are "subrecipients", and it could be confusing if we call the same thing "subrecipient" one place and "recipient" somewhere else.